php/.env.example

```diff
@@ -0,0 +1,24 @@
+APP_NAME=Lumen
+APP_ENV=local
+APP_KEY=
+APP_DEBUG=true
+APP_URL=http://localhost
+APP_TIMEZONE=UTC
+
+LOG_CHANNEL=stack
+LOG_SLACK_WEBHOOK_URL=
+
+DB_CONNECTION=mysql
```

Comment:
what are you using database for?

Reply:
Default configs for lumen, I will try and remove them:
- https://github.com/laravel/lumen/blob/8.x/.env.example#L11-L16